src/deploy/dataconnect/prepare.ts

```diff
@@ -55,6 +58,37 @@ export default async function (context: any, options: Options): Promise<void> {
     filters,
   };
   utils.logLabeledBullet("dataconnect", `Successfully prepared schema and connectors`);
+  if (options.dryRun) {
+    utils.logLabeledBullet("dataconnect", "Checking for CloudSQL resources...");
+    return;
```

Comment:
Just checking -- does this need to return early? Since below, you pass `options.dryRun` into the `provisionCloudSql` call anyway?